.github/ISSUE_TEMPLATE/specreview_checklist.md

```diff
@@ -0,0 +1,65 @@
+# Spec Review Checklist
+
```

Comment:
What about filling in some of the preamble for filling in some of the fields of the Issue, like what Scott had... Maybe not labels?  But, we need something to make this easier to understand for the people creating these issues.

```
name: Specification Review Issue
about: Start a request for a specification creation/plan/release... lifecycle review
title: [MP Component] [ Version] Specification Review
labels: creation,plan,progress,release
assignees: ''
```